pkg/flink/internal/controller/statement_controller.go

```diff
@@ -40,6 +41,11 @@ func (c *StatementController) ExecuteStatement(statementToExecute string) (*type
 	c.createdStatementName = processedStatement.StatementName
 	processedStatement.PrintStatusMessage()
 
+	if !processedStatement.IsLocalStatement && processedStatement.ServiceAccount == "" && processedStatement.IdentityPool == "" {
+		utils.OutputWarnf(`Warning: to ensure that your statements run continuously, switch to using a service account instead of your user identity by running "SET '%s'='sa-123';". Otherwise, statements will stop running after 4 hours.`,
```

Comment:
Why `utils.OutputWarnf()` and not `log.CliLogger.Warnf()`? This is what all other commands use.

Reply:
These functions colorize the output message (warn messages will be yellow for example)

Reply:
done